.github/workflows/test-runner-uploadresults.yml

```diff
@@ -1,25 +1,53 @@
 # This workflow will build a Java project with Gradle
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
 
-name: Test Runner Integration Tests
+name: Run Tests
```

Comment:
If we have a lot of repeated boilerplate, we could almost justify using moustache templates and code-gen'ing the workflows.

Reply:
I have added your comment to this Google [doc](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit#) to track it for future enhancements discussion.